go.mod

```diff
@@ -29,9 +29,9 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kubernetes v1.23.0
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
-	sigs.k8s.io/yaml v1.2.0
+	sigs.k8s.io/yaml v1.3.0
 	stathat.com/c/consistent v1.0.0
-	volcano.sh/apis v1.5.0-beta.0.0.20220427084719-22d3dd007a8f
+	volcano.sh/apis v0.0.0-20220630011627-e579d851e770
```

Comment:
The apis has been updated, please synchronize the latest commit number

Reply:
updated